### PR TITLE
Don't use Ninja for bloaty build

### PR DESCRIPTION
**Pull request by louwers**
It's not available on the default Ubuntu image...

### CODE REVIEW

**Comment by louwers** [comment]
@ntadej Sure I can also do that.

File: .github/workflows/pr-bloaty.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ jobs:
       - name: Compile Bloaty
         run: |
           git clone https://github.com/google/bloaty.git
-          cmake -B bloaty/build -G Ninja -S bloaty
+          cmake -B bloaty/build -S bloaty
```
Comment (ntadej) [inline .github/workflows/pr-bloaty.yml:39]: This will AFAIK build serially which we probably do not want.

**Review by ntadej — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-bloaty.yml (hunk 1 above)
Comment (louwers) [reply .github/workflows/pr-bloaty.yml]: Fixed.

**Review by ntadej — APPROVED** [review]
We should probably unify actions between workflows, but fine with me for now.

**Pull request merged**